scripts/node.sh

```diff
@@ -143,7 +143,8 @@ case "${BLSKEYFILE}" in
          BLSKEYFILE="${f}"
          ;;
       *)
-         err 69 "multiple key files found (${f}, ${BLSKEYFILE}); please use -k to specify"
+         [ "${f}" -ef "${BLSKEYFILE}" ] || \
```

Comment:
"-ef" is for testing hardlink. https://www.tldp.org/LDP/abs/html/fto.html . Is that what you want to test?

Reply:
Yes.